simpleclient_jetty/src/main/java/io/prometheus/client/jetty/JettyStatisticsCollector.java

```diff
@@ -0,0 +1,81 @@
+package io.prometheus.client.jetty;
+
+import io.prometheus.client.Collector;
+import org.eclipse.jetty.server.handler.StatisticsHandler;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Collect metrics from jettty's org.eclipse.jetty.server.handler.StatisticsHandler.
+ */
+public class JettyStatisticsCollector extends Collector {
+  private final StatisticsHandler statisticsHandler;
+  private static final List<String> EMPTY_LIST = new ArrayList<String>();
+
+  public JettyStatisticsCollector(StatisticsHandler statisticsHandler) {
+    this.statisticsHandler = statisticsHandler;
+  }
+
+  @Override
+  public List<MetricFamilySamples> collect() {
+    return Arrays.asList(
+            buildGauge("jetty_requests", "number of requests", statisticsHandler.getRequests()),
+            buildGauge("jetty_requests_active", "number of requests currently active", statisticsHandler.getRequestsActive()),
+            buildGauge("jetty_requests_active_max", "maximum number of active requests", statisticsHandler.getRequestsActiveMax()),
```

Comment:
The help string could make that clearer: "Maximum number of requests that have been active."